### DISCUSSION TITLE
Loss of data when resuming training?

### DISCUSSION THREAD

**Discussion by rmfrancisiii** [acids-ircam/RAVE#96] [Q&A] [answered]
Hello all,

I get the following warning each time I resume training. My understanding of what's happening here is extremely limited — is the fact that the various files listed here are not being reloaded a problem? How do I ensure that the files are reloaded for use in the subsequent training run? Thanks in advance!

"UserWarning: The dirpath has changed from 'runs/vocalset/rave/version_1/checkpoints' to 'runs/vocalset/rave/version_2/checkpoints', therefore `best_model_score`, `kth_best_model_path`, `kth_value`, `last_model_path` and `best_k_models` won't be reloaded. Only `best_model_path` will be reloaded."

**Comment by AffeZwei** [comment, marked as answer]
I also get this warning message every time, you don't need these files loaded every time because the necessary data for the next run is stored in `best_model_path`. That has been my experience so far, but I am also a newbie.

**Reply by rmfrancisiii** [reply]
Thanks, now that I've trained a couple of models they do seem to be turning out ok despite the warning.
